packages/ui-kit/src/components/AccessTokenProvider/AccessToken.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { fetchStorybookAccessToken, RelativeTimeRange, storybookCodeTemplate } from '../../helpers'
+import axiosInstance from '../../../../../app/storybook/src/axios'
+import { AccessTokenProvider } from './AccessTokenProvider'
+import { TimeSeries, Counter } from '..'
+
+const meta: Meta = {
+  title: 'PROVIDERS/AccessTokenProvider',
+  tags: ['pattern'],
+  parameters: {
+    imports: 'TimeSeries, Counter, AccessTokenProvider',
+    codeTemplate: storybookCodeTemplate
+  }
+}
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+async function fetchAccessToken() {
+  const accessToken = await fetchStorybookAccessToken(axiosInstance)
+
+  return accessToken ?? ''
+}
+
+export const AccessTokenProviderStory: Story = {
+  args: {
+    fetchToken: fetchAccessToken
+  },
+  argTypes: {
+    fetchToken: {
+      description: 'The function that fetches the access token.',
+      control: { type: 'function' }
+    },
+    accessToken: {
+      description: 'The access token that will be served to the components.',
+      control: { type: 'text' }
+    },
+    onAccessTokenExpired: {
+      description: 'The function that will be called when the access token expires.',
+      control: { type: 'function' }
+    }
+  },
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  render: (_) => {
+
+    const fetchToken = async () => {
```

Comment:
"fetchToken" function is a repetation. We do not need to create this function.
You can directly pass the "fetchAccessToken" to the provider.

So it is better if you rename it as "fetchToken" while directly passing it.